storage/innobase/log/log0log.cc

```diff
@@ -1251,49 +1242,36 @@ log_read_encryption()
 @param[in,out]	buf	log file header
 @param[in]	key	encryption key
 @param[in]	iv	encryption iv */
-static
-bool
-log_file_header_fill_encryption(
-	byte*		buf,
-	byte*		key,
-	byte*		iv)
-{
-	byte		encryption_info[ENCRYPTION_INFO_SIZE_V2];
+static bool
+log_file_header_fill_encryption(byte *buf, byte *key, byte *iv) {
+	byte encryption_info[ENCRYPTION_INFO_SIZE_V2];
 
-	if (!fsp_header_fill_encryption_info(key,
-				      iv, encryption_info)) {
+	if (!fsp_header_fill_encryption_info(key, iv, encryption_info)) {
 		return(false);
 	}
 
-	ut_ad(LOG_HEADER_CREATOR_END + ENCRYPTION_INFO_SIZE_V2
-	      < OS_FILE_LOG_BLOCK_SIZE);
+	ut_ad(LOG_HEADER_CREATOR_END + ENCRYPTION_INFO_SIZE_V2 <
+	      OS_FILE_LOG_BLOCK_SIZE);
 
-	memcpy(buf + LOG_HEADER_CREATOR_END,
-	       encryption_info,
+	memcpy(buf + LOG_HEADER_CREATOR_END, encryption_info,
 	       ENCRYPTION_INFO_SIZE_V2);
 
 	return(true);
 }
 
-static
-bool
-log_file_header_fill_encryption(
-	byte*		buf,
-	ulint           key_version,
-	byte*		iv)
-{
-	byte		encryption_info[ENCRYPTION_INFO_SIZE_V2] = {};
+static bool
+log_file_header_fill_encryption(byte *buf, ulint key_version, byte *iv) {
+	byte encryption_info[ENCRYPTION_INFO_SIZE_V2] = {};
 
 	if (!fsp_header_fill_encryption_info(key_version, iv,
-				      encryption_info)) {
+					     encryption_info)) {
```

Comment:
Is the indentation correct? (tabs followed by space for alignment)

Reply:
Yes, it's exactly at the right position with this.